docs/content/development/extensions-core/druid-kerberos.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: doc_page
+---
+
+# Druid-Kerberos
+
+Druid Extension to enable Authentication for Druid Nodes using Kerberos.
+This extension adds AuthenticationFilter which is used to proect HTTP Endpoints using the simple and protected GSSAPI negotiation mechanism [SPNEGO](https://en.wikipedia.org/wiki/SPNEGO). 
+Make sure to [include](../../operations/including-extensions.html) `druid-kerberos` as an extension.
+
+
+## Configuration
+
+|Property|Possible Values|Description|Default|
+|--------|---------------|-----------|-------|
+|`druid.authentication.type`|kerberos||Must be set to 'kerberos' to enable kerberos authetication.|empty|
```

Comment:
do we expect to have another type withing the same module  ? not sure if this is needed i propose to turn it into a switch to turn it on/off `druid.authentication.kerberos=true` by default.